src/docker/tools/index.go

```diff
@@ -61,14 +61,16 @@ func GetExitCode(containerName string) (int, error) {
 }
 
 // GetRenderedVolumes returns the rendered paths to the given volumes
-func GetRenderedVolumes(volumes []string, appName string, role string, homeDir string) ([]string, error) {
-	dataPath := path.Join(homeDir, ".exosphere", appName, role, "data")
+func GetRenderedVolumes(volumes []string, appDir string, role string) ([]string, error) {
+	appRelativePath := path.Join(".exosphere", "data", role)
+	realPath := path.Join(appDir, appRelativePath)
+	renderedPath := path.Join("${APP_PATH}", appRelativePath)
```

Comment:
If `appDir` is already something like `Users/charlie/exosphere/space-tweet`, and `${APP_PATH}` is also `Users/charlie/exosphere/space-tweet`, won't this line combine them into `Users/charlie/exosphere/space-tweet/Users/charlie/exosphere/space-tweet`? That seemed to be the behavior when I tested it out here: https://golang.org/pkg/path/#Join

Reply:
This is joining `${APP_PATH}` with `path.Join(".exosphere", "data", role)`.

Reply:
I'll rename `appRelativePath` to `dataRelativePath`